back/src/domain/convention/ports/ConventionQueries.ts

```diff
@@ -20,8 +21,9 @@ export interface ConventionQueries {
   getConventionById: (
     id: ConventionId,
   ) => Promise<ConventionReadDto | undefined>;
-  getAllConventionsForThoseEndingThatDidntReceivedAssessmentLink: (
+  getAllConventionsForThoseEndingThatDidntGoThroughSendingTopic: (
```

Comment:
proposition de renommage, car là on mélange du métier et du tech:
```suggestion
  getAllConventionsForThoseEndingThatDidntGoThrough: (
```